packages/lib/src/components/internal/Button/Button.stories.tsx

```diff
@@ -40,7 +42,7 @@ const coreProps = {
             confirmPreauthorization: 'Confirm preauthorization'
         }
     }),
-    resources: global.resources
+    resources: new Resources('https://checkoutshopper-test.cdn.adyen.com/checkoutshopper/')
```

Comment:
Is there a way for us not to hardcode this value? Maybe by using the environment variables